### ISSUE TITLE
[Hold - Browser Resizing - #13971] App becomes unresponsive when the window is resized to a smaller view while uploading an attachment

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13945]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open NewDot and login
2. Search for a contact with which there was no chat before
3. Click on + icon
4. Click Add attachment
5. Choose any attachment
6. Resize the window to a smaller view
7. Click on send button or Press esc key

## Expected Result:
Able to attach the image or should be able to cancel when pressing esc key

## Actual Result:
App becomes unresponsive

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?

- [ ] MacOS / Chrome / Safari

**Version Number:** v1.2.47-0
**Reproducible in staging?:** y 
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/210280623-b978f761-838d-4bf4-82c1-98d275f61c06.mp4

https://user-images.githubusercontent.com/43996225/210280642-9ca11ae6-5e0d-4417-8433-10694b32afa8.mov

**Expensify/Expensify Issue URL:**
**Issue reported by:** @adeel0202 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1672648703766349

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by trjExpensify** [comment]
@michaelhaxhiu putting [this thread](https://expensify.slack.com/archives/C01SKUP7QR0/p1672783427814329) on your radar and I've created [this parent issue](https://github.com/Expensify/App/issues/13971). We should tackle all of these browser resizing issues holistically and need someone to own that plan.

**Comment by michaelhaxhiu** [comment]
Thanks! Looking now.

**Comment by michaelhaxhiu** [comment]
Still holding

**Comment by michaelhaxhiu** [comment]
Still held

**Comment by michaelhaxhiu** [comment]
Still held.

**Comment by michaelhaxhiu** [comment]
held

**Comment by MelvinBot** [comment]
@michaelhaxhiu Huh... This is 4 days overdue. Who can take care of this?

**Comment by MelvinBot** [comment]
@michaelhaxhiu Huh... This is 4 days overdue. Who can take care of this?

**Comment by MelvinBot** [comment]
@michaelhaxhiu Huh... This is 4 days overdue. Who can take care of this?

**Comment by MelvinBot** [comment]
@michaelhaxhiu Still overdue 6 days?! Let's take care of this!

**Comment by michaelhaxhiu** [comment]
Changing to weekly since the parent issue is `monthly`, just to be conservative. Still held.

**Comment by michaelhaxhiu** [comment]
Breathe, melvin.

**Comment by michaelhaxhiu** [comment]
Held on nav still

**Comment by michaelhaxhiu** [comment]
Waiting still

**Comment by michaelhaxhiu** [comment]
held

**Comment by michaelhaxhiu** [comment]
Still held on https://github.com/Expensify/App/issues/13971

**Comment by michaelhaxhiu** [comment]
Note: I'm preparing to go OOO for ~2 weeks, and leaving this Monthly because I doubt it'll get finished while I'm gone. When I return I'll oversee it.

**Comment by michaelhaxhiu** [comment]
Back from OOO, but leaving `Monthly` since the tracking GH it's blocked on is `Monthly` too.

**Comment by tjferriss** [comment]
We're still on hold and waiting on the linked issue which itself is not being prioritized https://github.com/Expensify/App/issues/13971#issuecomment-1777736670

**Comment by tjferriss** [comment]
As far as I know this is still on hold for https://github.com/Expensify/App/issues/13971. @michaelhaxhiu do you want to take this one back? I've left the bug-zero team so I'd really appreciate it.

**Comment by michaelhaxhiu** [comment]
Not a priority yet.

**Comment by JmillsExpensify** [comment]
We don't plan on prioritizing browser resizing anytime soon. We're more focused on migrating our existing customers from our old platform to our new platform, which will minimally take the remainder of this year. We can open up this issue in the future if/when this becomes a priority.
